library/src/morpheus-master/sass/modules/tool/assignments/_assignments.scss

```diff
@@ -465,6 +470,40 @@
 		color: #9E9E9E;
 	}
 
+	.contentReviewDisclosure {
+		position: relative;
+		display: flex;
+		margin-right: $standard-space;
+		align-items: baseline;
+	}
+
+	.contentReviewDisclosure:focus {
+		outline-offset: -1px;
+	}
+
+	.contentReviewMessage {
+		background-color:#fff;
+		position: relative;
+		border: 1px solid;
+		border-radius: 5px;
+		margin: 7px 24px 4px 24px;
+		padding: 4px 8px;
+		background-color: #cef;
```

Comment:
This colour should probably be extracted out to an assignments variable. I see that it is used elsewhere in Assignments too, so the variable could be used there too.